src/utils/smartContract.ts

```diff
@@ -1,6 +1,6 @@
 import { contractAddress } from 'config';
 import json from 'contracts/ping-pong.abi.json';
-import { AbiRegistry, Address, SmartContract } from './sdkDappCore';
+import { AbiRegistry, Address, SmartContract } from 'utils/sdkCore';
```

Comment:
import { AbiRegistry, Address, SmartContract } from 'utils/sdkCore'; in a new PR